doc/nrf/ug_radio_fem.rst

```diff
@@ -127,12 +129,14 @@ This feature uses a model to compensate the FEM gain for the following external
 The model assumes that the FEM supply voltage is constant.
 To provide the value of this voltage to the MPSL subsystem, use the :kconfig:option:`CONFIG_MPSL_FEM_POWER_VOLTAGE` option.
 
+This model is enabled by default, provided that nRF21540 GPIO/SPI is enabled and selected. Refer to :ref:`ug_radio_fem_nrf21540_spi_gpio` for instructions how to do that.
```

Comment:
In this case it would be nice to tell how to disable the model, if the user does not need it and for some reasons (for example scarcity of memory), he wants to disable it.